events-api/src/modules/events/events.service.ts

```diff
@@ -1,10 +1,26 @@
 import { EventModel } from "#/modules/events/events.model";
 import { EventsRepository } from "#/modules/events/events.repository";
+import { EventCreateDto } from "./dto/requests/events-create.dto";
 
 export class EventsService {
   static async getEvents(): Promise<EventModel[]> {
     const events = await EventsRepository.getAll();
 
     return events;
   }
+
+  static async createEvent(newEvent: EventCreateDto): Promise<void> {
+    await EventsRepository.create(newEvent);
+  }
+
+  static async updateEvent(
+    event: EventCreateDto,
+    id: string
+  ): Promise<null | unknown> {
+    return await EventsRepository.update(event, id);
+  }
+
+  static async deleteEvent(id: string): Promise<null | unknown> {
+    return await EventsRepository.delete(id);
```

Comment:
This method should return the deleted event data

Reply:
![image](https://github.com/user-attachments/assets/2ae34bdb-de57-4d0e-9a67-a09de9a75f1f)

Reply:
Again, you can just select a record before deleting. This limitation is not a blocker

Reply:
![image](https://github.com/user-attachments/assets/b2528c2d-ea92-44f3-814c-64f04b059866)

Reply:
Here you returning an array of records, please return a single record